### DISCUSSION TITLE
AdoptOpenJDK mac OS auto-updater

### DISCUSSION THREAD

**Discussion by ricounet67** [adoptium/adoptium-support#245] [Q&A] [answered]
Do you know if there is a project to create an auto-updater for macOS ?
It exists for Windows, and it could be very useful to have it on macOS. 
It will provide the same functionality as the Oracle control panel

**Comment by aahlenst** [comment, marked as answer]
> Do you know if there is a project to create an auto-updater for macOS ?

AdoptOpenJDK is available via [MacPorts](https://ports.macports.org/?search=openjdk&search_by=name) and [Homebrew](https://github.com/adoptopenjdk/homebrew-openjdk).

If anyone wants to make an updater for the standalone installer, https://sparkle-project.org/ is the right choice. Pull requests are welcome. We're happy to provide as much assistance as we can [on our Slack workspace](https://adoptopenjdk.net/slack.html), but as far as I know nobody has the necessary knowledge to integrate it.

> It will provide the same functionality as the Oracle control panel

What would that be good for?
